src/bundle/Resources/views/themes/admin/ui/edit_base.html.twig

```diff
@@ -2,6 +2,14 @@
 
 {% block main_container_class %}ibexa-main-container--edit-container{% endblock %}
 
+
+{% block side_column_class -%}
```

Comment:
As we spoke, you should use `ibexa-main-container--with-anchor-menu-items`

Reply:
Good remark!